### PR TITLE
Select on surface

### PR DESCRIPTION
**Pull request by DQLin**
Hi everyone, just merged my select-on-surface function with nathan's disney brdf. Approval requested.

### CODE REVIEW

**Comment by n8xm** [comment]
0. We might be better off creating a pull request directly onto my branch (`disney-brdf-load`). That way it is easier to see which changes are yours and which changes are mine. 
1. Picking is broken on Chromium on Linux. It works on Firefox on Linux but the teapot disappears after clicking between 3 and 10 times. On Windows, it works fine in Firefox. For some reason my Chrome on Windows is choking - can @n8vm test this?
2. Right now we aren't updating the sliders on the left when we pick.  
3. Ideally, I think we should be moving the light when we pick. The incident ray should point in the light direction. 
4. This is a very handy feature but it might not be the highest priority right now. After I am done loading Disney's `.brdfs` (which should be around today), I'm going to draft a planning document. Then we can decide whether to continue to fix picking or if we want to focus on some other feature.

**Comment by n8xm** [comment]
Update on priorities. I'd hate to see @DQLin 's work go to waste. My point is that if it takes a really long time to debug on Linux we may want to move on.

**Comment by n8vm** [comment]
BRDF's specular lobe isn't pointing towards the light direction. Any idea why this might be?

I might have messed up the selection by changing the aspect ratio of the gl canvases to take into account the additional controls. Could you take a look when you get a chance?

File: Styles/flexbox.css [hunk 1]
```diff
@@ -77,8 +77,7 @@ html, body {
 	}
 	#brdf-canvas {
 		width: 80vw;
-		height: 80vw;
-
+		height: 68.5714286vw;
```
Comment (n8xm) [inline Styles/flexbox.css]: 68.5714286 is a "magic number" that scares me a bit. Why exactly did we choose this number? There should at least be a comment.

**Pull request merged**